test/DisplayBoardTests.cpp

```diff
@@ -52,7 +52,7 @@ TEST_F(DisplayBoardTests, checkDrawingEmptyRowsForMaxOneElementInRow)
 
 TEST_F(DisplayBoardTests, checkDrawingTable)
 {
-    std::string drawB = DisplayBoard::drawBoard(1,1,1,1).str();
+    std::string drawB = drawBoard(1,1,1,1).str();
```

Comment:
This is in unnamed namespace... I see you've included cpp file. Please don't do that.
Just make drawBoard public by moving it outside unnamed namespace and test everything via this function.

Reply:
I wanted to ask about it. 
I didn't know how to test private methods, and I found this guide:
https://github.com/google/googletest/blob/master/googletest/docs/advanced.md#testing-private-code
there is an advice that one of method can be include of cpp files, it is only for test. 
What do you think about it?

Reply:
I am closing it, because it is outdated.